src/service/cid-api.js

```diff
@@ -178,4 +185,12 @@ export class GoogleCidApi {
   expiresIn_(time) {
     return this.win_.Date.now() + time;
   }
+
+  isReferrerProxyOrigin_() {
+    return isProxyOrigin(WindowInterface.getDocumentReferrer(this.win_));
```

Comment:
not sure i understand the comment, but this code is planned to be run on pub origin.